modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainer.java

```diff
@@ -1,35 +1,15 @@
 package org.testcontainers.containers;
 
-import org.influxdb.InfluxDB;
-import org.influxdb.InfluxDBFactory;
-import org.testcontainers.containers.wait.strategy.Wait;
-import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
 import org.testcontainers.utility.DockerImageName;
 
-import java.util.Collections;
-import java.util.Set;
-
 /**
- * See <a href="https://store.docker.com/images/influxdb">https://store.docker.com/images/influxdb</a>
+ * @deprecated instead use {@link InfluxDBContainerV1} for InfluxDB 1.x or {@link InfluxDBContainerV2} for InfluxDB 2.x instead
  */
-public class InfluxDBContainer<SELF extends InfluxDBContainer<SELF>> extends GenericContainer<SELF> {
-
-    public static final Integer INFLUXDB_PORT = 8086;
-
+@Deprecated
+public class InfluxDBContainer extends InfluxDBContainerV1 {
```

Comment:
To make it binary compatible, we need to keep the self typing on this class:
```suggestion
public class InfluxDBContainer<SELF extends InfluxDBContainer<SELF>> extends InfluxDBContainerV1 {
```